src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.DataConverter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataConverterImpl implements DataConverter {
+    public static final String SEPARATOR = ",";
```

Comment:
Avoid hardcoding values in your service implementations. Consider passing 'SEPARATOR' as a parameter or defining it in a utility class if it's shared across multiple classes.